rosette_api/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +31,6 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.7.3.0")]
-[assembly: AssemblyVersion("1.7.3.0")]
+// [assembly: AssemblyVersion("1.7.3.0")]
+[assembly: AssemblyVersion("1.7.3.0")]
```

Comment:
Is this version string required? How can we avoid modifying the file if we just want to release with a version bump?

Reply:
It needs to be something.  The publish script for C# takes the provided version and updates this file automaticallly.